src/Persistence.php

```diff
@@ -76,12 +76,41 @@ public function disconnect()
     {
     }
 
+    /**
+     * Given a field seed, return a feld object.
+     *
+     * @param string $type
+     *
+     * @throws \atk4\core\Exception
+     *
+     * @return Field
+     */
+    public function fieldFactory($seed = [])
```

Comment:
I still don't think this is needed in Persistence. See my other comment above.

Reply:
removed.